### PR TITLE
Document limitations for loading Draeger data

### PR DESCRIPTION
**Pull request by psomhorst**
~~#192~~ #193 and #217 explain two limitation of loading Draeger data. This documents these.

### CODE REVIEW

File: eitprocessing/datahandling/loading/__init__.py [hunk 1]
```diff
@@ -18,6 +18,12 @@ def load_eit_data(
 ) -> Sequence:
     """Load EIT data from path(s).
 
+    Current limitations:
+    - Dräger data is assumed to have a sample frequency of 20 Hz. The sample frequency should be provided while loading
```
[the remark this answers is not in the document]
Comment (psomhorst) [reply eitprocessing/datahandling/loading/__init__.py:22]: I've addressed both issues and updated the tests accordingly.

File: eitprocessing/datahandling/loading/draeger.py [hunk 2]
```diff
@@ -37,7 +37,9 @@ def load_from_single_path(
     if file_size % _FRAME_SIZE_BYTES:
         msg = (
             f"File size {file_size} of file {path!s} not divisible by {_FRAME_SIZE_BYTES}.\n"
-            f"Make sure this is a valid and uncorrupted Dräger data file."
+            "Currently this package does not support loading files containing "
+            "esophageal pressure or other non-standard data. "
+            "Make sure this is a valid and uncorrupted Dräger data file."
```
Comment (DaniBodor) [inline eitprocessing/datahandling/loading/draeger.py:42]: Is this clearer? Also is it worthwhile explicitly stating to re-save files excluding those data and trying again?

```suggestion
            f"<{path!s}> cannot be loaded due to an incompatible file size.\n"
            "Make sure this is a valid and uncorrupted Dräger data file."
            "Note that this package does not currently support loading"
            "files containing esophageal pressure or other non-standard data.\n"
```

File: eitprocessing/datahandling/loading/__init__.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (DaniBodor) [reply eitprocessing/datahandling/loading/__init__.py:22]: I think the top issue can be removed here.
the assumption no longer holds true, right? It also does not feel like a "limitation" any more.

**Comment by DaniBodor** [comment]
are you pointing this at main rather than develop on purpose? Seems unlikely given that you've rebased or merged the branch on develop.

**Review by DaniBodor — APPROVED** [review]

**Pull request merged**